avm/res/synapse/workspace/main.bicep

```diff
@@ -570,3 +578,20 @@ type customerManagedKeyType = {
   @description('Optional. User assigned identity to use when fetching the customer managed key. Required if no system assigned identity is available for use.')
   userAssignedIdentityResourceId: string?
 }?
+
+type adminType = {
+  @description('Required. Workspace active directory administrator type.')
+  administratorType: string
+
+  @description('Required. Login of the workspace active directory administrator.')
+  @secure()
+  login: string
+
+  @description('Required. Object ID of the workspace active directory administrator.')
+  @secure()
+  sid: string
+
+  @description('Required. Tenant ID of the workspace active directory administrator.')
```

Comment:
```suggestion
  @description('Optional. Tenant ID of the workspace active directory administrator.')
```

Reply:
Requires an update of the readme & template too

Reply:
i forgot to update the description. Now it's done